### ISSUE TITLE
Dapper out-dated on NuGet

### ISSUE THREAD

**Issue by GoogleCodeExporter** [itsmebhavin/dapper-dot-net#109]
```
I was looking up for ways to serialize TimeSpan with Dapper. I realized Dapper 
(the version I had anyways), didn't support this right away, since it was 
missing a line like this:


      SqlMapper.typeMap[typeof (TimeSpan)] = DbType.Time;

In SqlMapper; which I realized after some digging, and getting to an issue on 
this page.

Then I re-downloaded the code from GitHub and it's fixed in v1.8, and it was 
then that I realized, Dapper on NuGet is just v1.1, shouldn't it be up to date 
on nuget?

otherwise referencing it through NuGet is pretty much purposeless.

TL;DR: update NuGet to reference Dapper 1.8 instead of Dapper 1.1

<3 Dapper!
```

Original issue reported on code.google.com by `nicolasb...@gmail.com` on 20 Aug 2012 at 8:40
